webui/react/src/components/kit/Breadcrumb.tsx

```diff
@@ -1,6 +1,10 @@
 import { Breadcrumb as AntdBreadcrumb } from 'antd';
 import React, { ReactNode } from 'react';
 
+import BreadcrumbBar from 'components/BreadcrumbBar';
+
+export { BreadcrumbBar };
```

Comment:
nit: if you're going to import and re-export BreadcrumbBar here you might as well just move it into this file. Especially since it's pretty small after the refactor.